fs-storage/src/cli/write.rs

```diff
@@ -0,0 +1,72 @@
+use std::collections::HashMap;
+use std::env;
+use std::fs;
+use std::io;
+use std::path::PathBuf;
+use fs_storage::file_storage::FileStorage;
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() != 2 {
+        println!("Usage: {} <path_to_storage>", args[0]);
+        return;
+    }
+
+    let storage_path = &args[1];
+    if fs::metadata(storage_path).is_ok() {
+        println!("Storage already exists at {}", storage_path);
+        return;
+    }
+
+    if let Err(err) = fs::create_dir_all(storage_path) {
+        println!("Error creating storage directory: {}", err);
+        return;
+    }
+    println!("Storage directory created successfully at {}", storage_path);
+
+    let mut kv_pairs: HashMap<String, String> = HashMap::new();
+    loop {
+        println!("Enter a key-value pair (key=value), or enter 'done' to finish:");
+        let mut input = String::new();
+        io::stdin().read_line(&mut input).expect("Failed to read line");
+        let input = input.trim();
+
+        if input.eq_ignore_ascii_case("done") {
+            break;
+        }
+
+        let pair: Vec<&str> = input.splitn(2, '=').collect();
+        if pair.len() != 2 {
+            println!("Invalid input, key-value pair must be in the format 'key=value'");
+            continue;
+        }
+
+        let key = pair[0].trim().to_string();
+        let value = pair[1].trim().to_string();
+
+        kv_pairs.insert(key, value);
+    }
+
+    println!("Key-Value Pairs:");
+    for (key, value) in &kv_pairs {
+        println!("{}: {}", key, value);
+    }
+
+    if let Err(err) = write_to_file(kv_pairs, storage_path) {
+        println!("Error writing to file: {}", err);
```

Comment:
Since this a dev cli errors are unwrapped in the new implementation

Reply:
> Since this a dev cli errors are unwrapped in the new implementation

I second this.
I know it's a grey area in rust where people take sides in general. For our case (CLI), I think we should never panic

I never want to see this output as a user when running a CLI command
```bash
thread 'main' panicked at fs-storage/src/bin/cli.rs:54:18:
Failed to read JSON file: Os { code: 2, kind: NotFound, message: "No such file or directory" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
I prefer passing down the error or using `anyhow::Context` 
Here is a reference from the "Command Line Applications in Rust" book
https://rust-cli.github.io/book/tutorial/errors.html

What do you think?